queries/java/locals.scm

```diff
@@ -1,10 +1,36 @@
 ; CREDITS @maxbrunsfeld (maxbrunsfeld@gmail.com)
 
-(class_declaration
-  name: (identifier) @name) @class
+;(class_declaration
+;  name: (identifier) @name) @class
```

Comment:
@defintiion.class?

Reply:
ok

Reply:
that part is from tree-sitter/tree-sitter-java

Reply:
commented out